src/main/scala/com/redhat/et/silex/histogram/rdd/rdd.scala

```diff
@@ -0,0 +1,148 @@
+/*
+ * This file is part of the "silex" library of helpers for Apache Spark.
+ *
+ * Copyright (c) 2015 Red Hat, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.c
+ */
+
+package com.redhat.et.silex.histogram.rdd
+
+import scala.language.implicitConversions
+import scala.reflect.ClassTag
+import scala.collection.mutable
+
+import org.apache.spark.AccumulableParam
+import org.apache.spark.rdd.RDD
+
+private object details {
```

Comment:
One other note; I usually make objects called `implicits` to hold implicit conversions too, but scalastyle complains about lowercase object names.  We'll need to keep that in mind because I'd like to start running scalastyle as part of our tests soon.